cmd/sealer/cmd/cluster/run.go

```diff
@@ -218,6 +222,38 @@ func runWithClusterfile(clusterFile string, runFlags *types.RunFlags) error {
 	return createNewCluster(imageEngine, cf, imageSpec, runFlags.Mode)
 }
 
+func getClusterInstallInfo(labels map[string]string) clusterruntime.InstallInfo {
```

Comment:
i agree with you that we need this installerinfo, while i think ,could we move those logic to installer.go, cmd no need to care about cluster-runtime info.

Reply:
Fine, I also think so, run module is no need to care Cluster runtime logic.